tools/fetch-sponsors.js

```diff
@@ -402,6 +457,7 @@ async function fetchGitHubSponsors() {
 	] = await Promise.all([
 		fetchOpenCollectiveData(),
 		fetchGitHubSponsors(),
+		fetchThanksDevData(),
```

Comment:
```suggestion
		fetchThanksDevData(),
```
The result of fetchThanksDevData() is not assigned to any variable or used in the destructuring. not sure if you are waiting for the confirmation.